### DISCUSSION TITLE
1.2.2 Display Filename During Host Print not working

### DISCUSSION THREAD

**Discussion by NoQuarter89** [Jyers/Marlin#303] [Q&A] [answered]
I compiled and flashed the 1.2.2 build and printing from Octoprint still displays "Host Print" on my screen. Is there something I need to change in configuration.h or adv.h? I used the configuration templates for BLtouch3x3HS.

**Comment by joeclifford** [comment, marked as answer]
You need to add more settings to octoprint.

https://github.com/Jyers/Marlin/issues/238#issuecomment-798940245

 
>All you need to do it add `M117 <F>{{ event.name }}` after M75 in your octoprint "Before print job starts" gcode script.

**Reply by Schnabulation** [reply]
Can confirm this is working just fine.

**Reply by NoQuarter89** [reply]
Thank you both

**Reply by LABMONITOR** [reply]
Just wanted to contribute something here -- if you're like me and sometimes have unreasonably long filenames, you can strip the ".gcode" extension on the name by adding

```M117 <F>{{ event.name[:-6] }}```

... to make that header look a bit cleaner since it would always end with a ".gcode" anyway (or at least in my case). This only works on that M117 message that replaces "Host Print" and doesn't affect anywhere else that the filename is used (e.g., the actual file, other plugins that use it, etc)

**Comment by Jyers** [comment]
Upon a user suggestion I have also created this to help new users: https://github.com/Jyers/Marlin/wiki/OctoPrint-Settings

**Reply by NoQuarter89** [reply]
Thanks!

**Reply by GuidoHendriks** [reply]
Maybe there could be a reference to there's from the readme? I didn't find that and can imagine others wouldn't.

Appreciate your work. Are there any options for donating?

**Reply by Schnabulation** [reply]
@Jyers I would also recommend to adjust the README.md. It's pretty easy and you can just rewrite the file in the root (but I'm sure you already know this...)

**Reply by Schnabulation** [reply]
> Are there any options for donating?

Top right corner: Click the button "Sponsor".

**Reply by Jyers** [reply]
Yeah, I made the wiki partially so I could avoid modifying the readme. But perhaps I should append a bit to the top describing where to find import information. That way I can also send people strait here instead of to the reddit post first.

**Reply by NoQuarter89** [reply]
@Jyers apologies if this is the incorrect thread to ask, but in the wiki you mention you like both M73 Progress and Detailed Progress. Do you actually need both? Or can I just use Detailed Progress?
